git/repo/base.py

```diff
@@ -697,6 +697,19 @@ def _get_untracked_files(self, *args, **kwargs):
         finalize_process(proc)
         return untracked_files
 
+    def get_ignored(self, *paths):
+        """Checks if paths are ignored via .gitignore
+        Doing so using the "git check-ignore" method.
+
+        :param paths: List of paths to check whether they are ignored or not
+        :return: sublist of ignored paths
+        """
+        try:
+            proc = self.git.check_ignore(*paths)
+        except GitCommandError:
+            return []
+        return proc.replace("\\\\", "\\").replace('"', "").split("\n")
```

Comment:
I believe this is not sufficient to undo the [c-style quoting](https://github.com/git/git/blob/9bc233ae1cf19a49e51842c7959d80a675dbd1c0/builtin/check-ignore.c#L44:L44) that git applies to the output, even though superficially it may seem to be the case.
In case you think it's worth improving this, here [is the code to unquote in C](https://github.com/git/git/blob/9bc233ae1cf19a49e51842c7959d80a675dbd1c0/quote.c#L394:L394) and [here it is in Rust](https://github.com/Byron/gitoxide/blob/e355b4ad133075152312816816af5ce72cf79cff/git-odb/src/alternate/unquote.rs#L22).
If not, just let me know and we go ahead with the merge, fixing it when the first bug report comes in.